Fix fmt-related build errors when in --std=c++20 mode

Copied from https://github.com/dreal/dreal4/pull/283.

When we upgrade to a version of dReal that already incorporates this
patch, the patching process will fail ("already applied") at which
point we can remove this file.

--- dreal/smt2/sort.cc
+++ dreal/smt2/sort.cc
@@ -35,7 +35,7 @@ Sort ParseSort(const string& s) {
   if (s == "Binary") {
     return Sort::Binary;
   }
-  throw DREAL_RUNTIME_ERROR("{} is not one of {Real, Int, Bool}.", s);
+  throw DREAL_RUNTIME_ERROR("{} is not one of {{Real, Int, Bool}}.", s);
 }
 
 ostream& operator<<(ostream& os, const Sort& sort) {
diff --git a/dreal/util/ibex_converter.cc b/dreal/util/ibex_converter.cc
index 4e107b0f..ae6e18ad 100644
--- dreal/util/ibex_converter.cc
+++ dreal/util/ibex_converter.cc
@@ -138,7 +138,7 @@ const ExprNode* IbexConverter::VisitVariable(const Expression& e) {
       oss << v << " ";
     }
     oss << ".";
-    throw DREAL_RUNTIME_ERROR(oss.str());
+    throw DREAL_RUNTIME_ERROR("{}", oss.str());
   }
   return it->second;
 }
